<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1168</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Forward">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Forward</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Backward">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Backward</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Leftward">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Leftward</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Rightward">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Rightward</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Upward">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Upward</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Downward">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>120</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Downward</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>351</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QPushButton" name="xdown">
       <property name="text">
        <string>xdown</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="xup">
       <property name="text">
        <string>xup</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="yup">
       <property name="text">
        <string>yup</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="ydown">
       <property name="text">
        <string>ydown</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="zup">
       <property name="text">
        <string>zup</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="zdown">
       <property name="text">
        <string>zdown</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="Port">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>60</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>/dev/ttyUSB0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>/dev/ttyACM0</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="Baudrate">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>120</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="currentText">
     <string>9600</string>
    </property>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>398</x>
      <y>64</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>120</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Baudrate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Connect">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>180</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>240</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Consoale</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>469</x>
      <y>30</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="TextShower">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>361</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="ShowInHex">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Show in Hex?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Pause">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="claw1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Grab</string>
    </property>
   </widget>
   <widget class="QPushButton" name="claw2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Grab</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Claw1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Claw2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>0</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Next Move</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Joint1Add">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>20</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Joint1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Joint2Add">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>60</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Joint2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Joint3Add">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>100</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Joint3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Joint4Add">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>140</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Joint4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Joint5Add">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>180</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Joint5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Joint6Add">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>220</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Joint6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Joint1D">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>20</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Joint1D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Joint3D">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>100</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Joint3D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Joint2D">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>60</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Joint2D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Joint6D">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>220</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Joint6D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Joint4D">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>140</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Joint4D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Joint5D">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>180</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Joint5D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NextMove">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>260</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>NextMove</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1168</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
